homeassistant/components/alarm_control_panel/envisalink.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, partition_number, alarm_name, code, panic_type, info,
     def _update_callback(self, partition):
         """Update HA state, if needed."""
         if partition is None or int(partition) == self._partition_number:
-            self.hass.async_add_job(self.update_ha_state)
+            self.hass.async_add_job(self.async_update_ha_state())
```

Comment:
I change that. I think if that device realy async, the maintainer should that migrate complete to async (async_setup_platform, ect)